### PR TITLE
Create radar page

### PR DESCRIPTION
**Pull request by vladimir-sviridenko**

### CODE REVIEW

File: src-fe/src/app/libs/radar-view/components/create-radar-page/create-radar-page.component.ts [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+	}
+
+	public updateLocalConfig(config: RadarConfig): void {
+		this.config = config;
+	}
+
+	private initRadarConfig(): void {
+		this.config = {
+			name: '',
+			csv: '',
```
Comment (clain-atomi) [inline src-fe/src/app/libs/radar-view/components/create-radar-page/create-radar-page.component.ts:73]: not sure we need to initialize csv from default

File: src-fe/src/app/libs/radar-view/components/create-radar-page/create-radar-page.component.ts [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+			type: ButtonType.Outlined,
+		};
+
+		const applyButton: ButtonModel = {
+			label: 'Apply radar data',
+			callback: () => {
+				const radarId: string = this.createRadar();
+				this.updates$.pipe(ofType(RadarViewActionTypes.LoadRadarsSuccess), take(1)).subscribe(() => {
+					this.router.navigateByUrl(`/radars/${radarId}`);
+				});
```
Comment (clain-atomi) [inline src-fe/src/app/libs/radar-view/components/create-radar-page/create-radar-page.component.ts:113]: this must be in effect

**Review by clain-atomi — CHANGES_REQUESTED** [review]

File: src-fe/src/app/libs/radar-view/components/create-radar-page/create-radar-page.component.ts [hunk 3]
```diff
@@ -0,0 +1,122 @@
...
+			disabled: false,
+			type: ButtonType.Flat,
+		};
+
+		this.buttons = [downloadButton, uploadButton];
+		this.extraButtons = [cancelButton, applyButton];
+	}
+
+	private createRadar(): string {
+		const radarId: string = v4();
```
Comment (clain-atomi) [inline src-fe/src/app/libs/radar-view/components/create-radar-page/create-radar-page.component.ts:118]: better move `radarId` as public property, and init it inside `ngOnInit()`

**Review by clain-atomi — CHANGES_REQUESTED** [review]

File: src-fe/src/app/libs/radar-view/components/create-radar-page/create-radar-page.component.ts (hunk 1 above)
Comment (vladimir-sviridenko) [reply src-fe/src/app/libs/radar-view/components/create-radar-page/create-radar-page.component.ts]: we need it to create empty radar if csv was not provided

**Review by clain-atomi — APPROVED** [review]

**Pull request merged**